app/src/main/res/drawable/background_gradien.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape  xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.background_gradien` appears to be unused
 Unused resources make applications larger and slow down builds.